test/tests/reporter/GriddedDataReporter/dummy.i

```diff
@@ -0,0 +1,33 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+[]
```

Comment:
Change the name of this to griddedData.i or something

Reply:
done